openmmtools/alchemy.py

```diff
@@ -1167,10 +1483,11 @@ def _alchemically_modify_PeriodicTorsionForce(reference_force, alchemical_region
         """
         # Don't create a force if there are no alchemical torsions.
         if len(alchemical_region.alchemical_torsions) == 0:
-            return [copy.deepcopy(reference_force)]
+            return {'': [copy.deepcopy(reference_force)]}
```

Comment:
I don't understand the need for the change do key-less dict here?

Reply:
Each `alchemically_modify_X()` method now has a different signature. The key of the returned `dict` is the parameter that controls that force so that we know upstream in which force group to put it.